git_machete/client.py

```diff
@@ -2110,7 +2110,7 @@ def create_github_pr(
         description: str = utils.slurp_file_or_empty(description_path)
 
         ok_str = ' <green><b>OK</b></green>'
-        print(fmt(f'Creating a {"draft " if opt_draft else ""}PR from `{head}` to `{base}`...'), end='', flush=True)
+        print(fmt(f'Creating a {"draft " if opt_draft else ""}PR from `{head}` to `{base}`...'), flush=True)
         pr: GitHubPullRequest = create_pull_request(org, repo, head=head, base=base, title=commits[0].subject,
                                                     description=description, draft=opt_draft)
         print(fmt(f'{ok_str}, see <b>{pr.html_url}</b>'))
```

Comment:
Hmmm how does the output look now in the happy path? I suppose `OK` is moved to another line, which looks weird

Reply:
@PawelLipski **Now:**
debug: 
```
__fire_github_api_request(method=POST, path=/repos/amalota/test_repo/pulls, token=***, request_body={'head': 'g', 'base': 'd', 'title': 'cd', 'body': '', 'draft': False}): firing a POST request to https://api.github.com/repos/amalota/test_repo/pulls with a bearer token and request body {"head": "g", "base": "d", "title": "cd", "body": "", "draft": false}
Creating a PR from `g` to `d`...
__fire_github_api_request(method=POST, path=/repos/amalota/test_repo/issues/10/assignees, token=***, request_body={'assignees': ['amalota']}): firing a POST request to https://api.github.com/repos/amalota/test_repo/issues/10/assignees with a bearer token and request body {"assignees": ["amalota"]}
OK, see https://github.com/amalota/test_repo/pull/10
Adding `amalota` as assignee to PR #10...OK
```

normal
```
Creating a PR from `g` to `d`...
OK, see https://github.com/amalota/test_repo/pull/11
Adding `amalota` as assignee to PR #11...OK
```

**before:**
```
__fire_github_api_request(method=POST, path=/repos/amalota/test_repo/pulls, token=***, request_body={'head': 'g', 'base': 'd', 'title': 'cd', 'body': '', 'draft': False}): firing a POST request to https://api.github.com/repos/amalota/test_repo/pulls with a bearer token and request body {"head": "g", "base": "d", "title": "cd", "body": "", "draft": false}
Creating a PR from `g` to `d`...OK, see https://github.com/amalota/test_repo/pull/13
__fire_github_api_request(method=POST, path=/repos/amalota/test_repo/issues/13/assignees, token=***, request_body={'assignees': ['amalota']}): firing a POST request to https://api.github.com/repos/amalota/test_repo/issues/13/assignees with a bearer token and request body {"assignees": ["amalota"]}
Adding `amalota` as assignee to PR #13...OK
```

I would rollback to what was before but changed: `, end='' -> ", flush=True) which results in:

```
__fire_github_api_request(method=POST, path=/repos/amalota/test_repo/pulls, token=***, request_body={'head': 'g', 'base': 'd', 'title': 'cd', 'body': '', 'draft': False}): firing a POST request to https://api.github.com/repos/amalota/test_repo/pulls with a bearer token and request body {"head": "g", "base": "d", "title": "cd", "body": "", "draft": false}
Creating a PR from `g` to `d`...  -> OK, see https://github.com/amalota/test_repo/pull/28
__fire_github_api_request(method=POST, path=/repos/amalota/test_repo/issues/28/assignees, token=***, request_body={'assignees': ['amalota']}): firing a POST request to https://api.github.com/repos/amalota/test_repo/issues/28/assignees with a bearer token and request body {"assignees": ["amalota"]}
Adding `amalota` as assignee to PR #28... -> OK
```

Reply:
Okay! proceed then